src/openrct2/sprites.h

```diff
@@ -840,6 +840,14 @@ enum {
     
     SPR_G2_EYEDROPPER = SPR_G2_BEGIN + 123,
 
+    SPR_G2_CHAR_BEGIN = SPR_G2_BEGIN + 124,
```

Comment:
I think we ought to make constants for each of the different fonts (small, normal, tiny).